app/helpers/transition-to.js

```diff
@@ -0,0 +1,12 @@
+import Helper from '@ember/component/helper';
+import { inject as service } from '@ember/service';
+
+export default Helper.extend({
+  router: service(),
+
+  compute([routeName, ...params]) {
```

Comment:
This is pretty cool. Still is this not overly complicated?
Is there a way to use {{link-to}} instead of this helper?

Not necessarily for or against just want to discuss.

Reply:
Since I've replaced usage of this with a `link-to`, I've removed the helper and related test file.